clang/test/CodeCompletion/crash-command-line-source-loc-zero.cpp

```diff
@@ -0,0 +1,12 @@
+// Related to #139375
```

Comment:
I don't why some regression tests are named `crash-*`, but I would not name this `crash-*`.

I'd just name this source-loc-zero.cpp and change this comment to `// Regression test for #139375`